readme.md

```diff
@@ -1,7 +1,7 @@
 # Search bar for Airbnb
-Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+Replace `Constantin-Haranskiy` with your Github username and copy the links to Pull Request description:
+- [DEMO LINK](https://Constantin-Haranskiy.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://Constantin-Haranskiy.github.io/layout_search-bar-airbnb/report/html_report/)
```

Comment:
Do not forget to add this links to the PR description

Reply:
Ok, am I need to remake PR?